std-bits/table/src/main/java/org/enso/table/data/column/operation/unary/IsFiniteOperation.java

```diff
@@ -0,0 +1,72 @@
+package org.enso.table.data.column.operation.unary;
+
+import java.util.BitSet;
+import org.enso.table.data.column.builder.BoolBuilder;
+import org.enso.table.data.column.builder.Builder;
+import org.enso.table.data.column.operation.UnaryOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemAggregator;
+import org.enso.table.data.column.storage.*;
+
+public class IsFiniteOperation extends AbstractUnaryBooleanOperation {
+  public static final String NAME = "is_finite";
+  public static final UnaryOperation INSTANCE = new IsFiniteOperation();
+
+  private IsFiniteOperation() {
+    super(NAME, true);
+  }
+
+  @Override
+  public boolean canApply(ColumnStorage storage) {
+    return storage.getType().isNumeric();
+  }
+
+  @Override
+  public ColumnStorage apply(
+      ColumnStorage storage, MapOperationProblemAggregator problemAggregator) {
+    if (storage instanceof ColumnLongStorage
+        && storage instanceof ColumnStorageWithNothingMap withNothingMap) {
+      // For a Column of Longs where we have the Nothing map, we can produce result immediately.
+      return new BoolStorage(
+          new BitSet(), withNothingMap.getIsNothingMap(), (int) storage.getSize(), true);
+    }
+
+    return super.apply(storage, problemAggregator);
+  }
+
+  @Override
+  protected void applyLong(
+      ColumnLongStorage longStorage,
+      Builder builder,
+      MapOperationProblemAggregator problemAggregator) {
+    var boolBuilder = (BoolBuilder) builder;
+    UnaryOperation.applyOverLongStorage(
+        longStorage, true, builder, (isNothing, value) -> boolBuilder.appendBoolean(!isNothing));
```

Comment:
@radeusgd I wonder if `isNothing` is always false here (and elsewhere in this file)? It seemed that way to me.

Reply:
Hmmm, right - if you set `nothingUnchanged` parameter to `true` then indeed the missing values will be skipped and your callback will not be called for them. So indeed your assumption seems to be right.

But I would be careful about it. I think the way the current code looks is OK. If you'd like to replace `boolBuilder.appendBoolean(!isNothing)` with `boolBuilder.appendBoolean(true)`, then I'd probably also add an `assert !isNothing;` in there, so that at least in the `-ea` runs we validate the assumption.

Reply:
Ok, I was just wondering if I was missing a case. I'll leave it like this, since it expresses the intention.